telesign/phoneid.py

```diff
@@ -23,3 +31,78 @@ def phoneid(self, phone_number, **params):
         """
         return self.post(PHONEID_RESOURCE.format(phone_number=phone_number),
                          **params)
+
+    def _execute(self, method_function, method_name, resource, **params):
+        resource_uri = "{api_host}{resource}".format(api_host=self.api_host, resource=resource)
+
+        json_fields = json.dumps(params)
+
+        headers = self.generate_telesign_headers(self.customer_id,
+                                                 self.api_key,
+                                                 method_name,
+                                                 resource,
+                                                 json_fields,
+                                                 user_agent=self.user_agent)
+
+        if method_name in ['POST', 'PUT']:
+            payload = {'data': json_fields}
+        else:
+            payload = {'params': json_fields}
+
+        response = self.Response(method_function(resource_uri,
+                                                 headers=headers,
+                                                 timeout=1000,
+                                                 **payload))
+
+        return response
+
+    @staticmethod
+    def generate_telesign_headers(customer_id, api_key, method_name, resource, json_fields, date_rfc2616=None,
```

Comment:
Same here. The point of the staticmethod is that customers can extract this in entirety and use it in their own application if they would prefer not to use the SDK. Would prefer to see these changes incorporated into the base class RestClient function.